src/httpParsing/model.ts

```diff
@@ -1,4 +1,16 @@
-// An empty object that contains all expected fields for a http request
+export type JSONValue = string | number | boolean | JSONObject | JSONArray;
+
+export type JSONObject = {
+  [x: string]: JSONValue;
+};
+
+export type JSONArray = Array<JSONValue>;
+
+export type postData = {
+  mimeType: string;
+  parsed?: JSONObject | JSONArray;
```

Comment:
I don't think we should do parsing here, but also, when you talk about JSON, by definition, this is `any`.

Reply:
agreed, removed parsing from this pr